cmd/neofs-ir/main.go

```diff
@@ -67,7 +67,7 @@ func main() {
 	c.Level = logLevel
 	c.Encoding = cfg.GetString("logger.encoding")
 	c.Sampling = nil
-	if term.IsTerminal(int(os.Stdout.Fd())) {
+	if term.IsTerminal(int(os.Stdout.Fd())) || cfg.GetBool("logger.timestamp") {
```

Comment:
I'd rather follow `logger.timestamp` if it's set. So that if it's explicitly `false` then tty detection logic shouldn't be used.